### ISSUE TITLE
`Ractive` not defined in playground

### ISSUE THREAD

**Issue by p3k** [ractivejs/ractive#3180]
## Description:

In certain situations the Ractive playground does not provide the `Ractive` variable.

## Versions affected:

Ractive playground at https://ractive.js.org

## Platforms affected:

Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/63.0.3239.84 Chrome/63.0.3239.84 Safari/537.36

## Reproduction:

1. Open a new browser window
1. Locate the [first tutorial](https://ractive.js.org/tutorials/hello-world/) on the Ractive website in the browser
1. Click on the “Ractive Playground” tab on the right side of the web page
1. Enter `Ractive` in the “Script” tab
1. In the developer console it will say `Uncaught ReferenceError: Ractive is not defined`

**Comment by ghost** [comment]
By default, the playground does not include Ractive (think of it as a fresh jsFiddle, no frameworks selected). The version of Ractive can be selected by clicking on the gear icon and selecting which version of Ractive in the External Scripts section.

Although I do agree that either of the following should be the case:
- The latest Ractive pre-selected.
- Have that dropdown available somewhere on the initial interface (although it would be very cramped).

**Comment by p3k** [comment]
OK, I probably feel a little bit misled by the name “Ractive playground” then… And I admit I did not even know there is an option to select different Ractive versions behind the gear icon.

I got confused after running various tutorials in the playground when “suddenly” `Ractive` was not defined. Of course, I would not have been able to tell if I opened the playground from within a tutorial or with the right tab.

Thus, I’d go +1 for the Ractive version pre-selected and available in the playground, no matter how it was activated.
